tests/infrastructure.py

```diff
@@ -42,7 +42,11 @@ def compare_tensor_to_golden(
         if assert_on_error:
             assert ret, f"PCC is {pcc} which is less than {required_pcc}"
 
-    atol = jnp.max(jnp.abs(tensor - golden))
+    atol = (
```

Comment:
If `jnp.max(jnp.abs(tensor - golden))` will work once https://github.com/tenstorrent/tt-xla/issues/73 is fixed, leave a comment here to revert this once fixed.

Reply:
It will not, those are separate issues (my apologies if I worded in wrongly). The test is skipped because of https://github.com/tenstorrent/tt-xla/issues/73, and I will enable it when it is fixed.

Reply:
if don't get why `jnp.max(jnp.abs(tensor - golden))` wouldn't work if `tensor` and `golden` were scalars. I just tried it

```
>>> def atol(a, b):
...     return jnp.max(jnp.abs(a-b))
... 
>>> atol(1, 2)
Array(1, dtype=int32)
>>> a = np.array([1, 2, 3])
>>> b = np.array([4, 5, 6])
>>> atol(a, b)
Array(3, dtype=int32)
```

Reply:
The issue was that the default test behaviour was putting both the golden and silicon output on silicon, leading to the `jnp` to be ran on our silicon, which lead to issues with scalars running on tt-metal. Changed the code to force the check to be ran on cpu and reverted the change.

Reply:
Wow, that's a big deal. Let's leave this thread open for @mrakitaTT to take a look.

Reply:
Discussed offline, turns out we need to be more explicit about where we put tensors and run graphs. Andrej will open separate task to streamline this.